### PR TITLE
Add /animelyrics command

### PR DESCRIPTION
**Pull request by colorfusion**
Add support to search for anime lyrics
because `oh yes, doki doki sunshine`.

### CODE REVIEW

File: test_app.py [hunk 1]
```diff
@@ -95,6 +95,14 @@ def test_lyrics4():
     app.test_mode()
     app.handle_command('/lyrics asdwqjwodjaosjdoqjwodjaosd by qiwjdoijasidjoqiwjdoiajsodijqwd', 0)
 
+def test_animelyrics():
+    app.test_mode()
+    app.handle_command('/lyrics renai circulation', 0)
```
Comment (res0nance) [inline test_app.py]: `/animelyrics`?

File: test_app.py [hunk 2]
```diff
@@ -95,6 +95,14 @@ def test_lyrics4():
...
+
+def test_animelyrics_no_lyrics():
+    app.test_mode()
+    app.handle_command('/lyrics omgwtfbbq', 0)
```
Comment (res0nance) [inline test_app.py]: same here

**Review by res0nance — CHANGES_REQUESTED** [review]

File: test_app.py (hunk 2 above)
Comment (colorfusion) [reply test_app.py]: Fixed

File: test_app.py (hunk 1 above)
Comment (colorfusion) [reply test_app.py]: Fixed

**Pull request merged**